traffic_ops/install/bin/_postinstall

```diff
@@ -737,7 +729,7 @@ def write_encryption_key(aes_key_location): # type: (str) -> None
 		"-base64",
 		"32"
 	)
-	if not exec_openssl("Generating an AES encryption key", *args):
+	if not exec_openssl("Generating an AES encryption key to %s" %aes_key_location, *args):
```

Comment:
The string modulo operator is outdated, this should use `.format()`.

Reply:
maybe I'm just old school 😎 

done